build.sbt

```diff
@@ -130,8 +130,8 @@ lazy val tokenizers = crossProject
   .in(file("scalameta/tokenizers"))
   .settings(
     publishableSettings,
-    description := "Scalameta APIs for tokenization and their baseline implementation",
-    libraryDependencies += "com.lihaoyi" %%% "scalaparse" % "0.4.2",
+    description := "Scala.meta's APIs for tokenization and its baseline implementation",
```

Comment:
We should keep the original description, we recently changed the spelling from "scala.meta" to "scalameta"